docs/source/userguide/export_post_processing.rst

```diff
@@ -140,49 +146,55 @@ Finally, you can add the :class:`festim.DerivedQuantities` object to the simulat
         ]
     )
 
-    my_model.exports = [my_derived_quantities, ....]
+    my_model.exports = [my_derived_quantities]
 
 
 The complete list of derived quantities can be found at: :ref:`Exports`.
 
 The data can be accessed in three different ways:
 - directly from the :class:`festim.DerivedQuantities` (plural) object:
```

Comment:
The PR looks fine to me. However, this part lacks an example on the [filter](https://festim.readthedocs.io/en/latest/api/exports.html#festim.DerivedQuantities.filter) method, isn't it? Probably, it might be added in a dedicated PR.

Reply:
Agreed! I'll open an issue